src/Button.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+function Button({ type, title, resetCount}) {
```

Comment:
Hier had ik zelf geen ReactComponent van gemaakt, omdat het niet voor een vermindering van code zorgt, of logica weg geabstraheerd. In feite is dit component nu gewoon een doorgeefluik.